src/LearnositySdk/Request/Init.php

```diff
@@ -140,6 +141,39 @@ public function __construct($service, $securityPacket, $secret, $requestPacket =
         $this->securityPacket['signature'] = $this->generateSignature();
     }
 
+    /**
+     * Adds metadata to request packet to enable telemetry and tracking. Request packet will be
+     * extended with following parameters:
+     * {
+     *   "meta": {
+     *     "sdk": {
+     *       "version": "v1.2.1",
+     *       "lang": "java",
+     *       "lang_version": "oracle-v1.9.0",
+     *       "platform": "linux",
+     *       "platform_version": "10"
```

Comment:
It might be worth updating this example to the real thing once we have it, so as not to have confusing docstrings. Not a job for now, though.

Reply:
Just to confirm - by real thing you mean real values?